### ISSUE TITLE
tumblr rips stall out with 'json is null' error

### ISSUE THREAD

**Issue by receptakill** [ripmeapp2/ripme#89]
* Ripme version:
* Java version:   18
* Operating system:   Win7 x64
* Exact URL you were trying to rip when the problem occurred:  https://iguanodont.tumblr.com/
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Quick and easy rip of a tumblr gallery

## Actual Behavior

Ripper stalls out immediately with the following errormsg:

```
Downloading http://api.tumblr.com/v2/blog/iguanodont.tumblr.com/posts/photo?api_key=JFNLu3CbINQjRdUvZibXW9VpSEVYYtiPJ86o8YmvgLZIoKyuNX&offset=0
Rip complete, saved to C:\Users\X\Desktop\rips2\tumblr_iguanodont.tumblr.com
Cannot invoke "org.json.JSONObject.getJSONObject(String)" because "json" is null
```

Logger entry is as follows:

```
Using api key: FQrwZMCxVnzonv90rgNUJcAk4FpnoS0mYuSuGYqIpM2cFgp9L4
Found album ripper: com.rarchives.ripme.ripper.rippers.TumblrRipper
Saved configuration to C:\Users\X\AppData\Local\ripme\rip.properties
Getting key queue in en_US value Queue
Getting key queue in en_US value Queue
Using api key: FQrwZMCxVnzonv90rgNUJcAk4FpnoS0mYuSuGYqIpM2cFgp9L4
Found album ripper: com.rarchives.ripme.ripper.rippers.TumblrRipper
Using album title 'tumblr_iguanodont.tumblr.com'
[+] Creating directory: ..\..
Set working directory to: C:\Users\X\Desktop\rips2\tumblr_iguanodont.tumblr.com
Initializing Main thread pool with 5 threads
Using api key: FQrwZMCxVnzonv90rgNUJcAk4FpnoS0mYuSuGYqIpM2cFgp9L4
Retrieving http://api.tumblr.com/v2/blog/iguanodont.tumblr.com/posts/photo?api_key=FQrwZMCxVnzonv90rgNUJcAk4FpnoS0mYuSuGYqIpM2cFgp9L4&offset=0
Trying to load cookies from config for api.tumblr.com
Trying to load cookies from config for tumblr.com
Got exception while running ripper:
java.lang.NullPointerException: Cannot invoke "org.json.JSONObject.getJSONObject(String)" because "json" is null
	at com.rarchives.ripme.ripper.rippers.TumblrRipper.handleJSON(TumblrRipper.java:248) ~[ripme-2.1.2-3-ea90b172.jar:2.1.2-3-ea90b172]
	at com.rarchives.ripme.ripper.rippers.TumblrRipper.rip(TumblrRipper.java:218) ~[ripme-2.1.2-3-ea90b172.jar:2.1.2-3-ea90b172]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.2-3-ea90b172.jar:2.1.2-3-ea90b172]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Waiting for threads to finish
   Rip completed!
Deleting empty directory C:\Users\X\Desktop\rips2\tumblr_iguanodont.tumblr.com
Getting key open in en_US value Open
```

**Comment by soloturn** [comment]
can you please try with a recent version?

**Comment by MacTerrassa** [comment]
I just tried the same link in the first message, and it gives me the same result. I'm using fully updated macos 14 Sonoma on an iMac M3, running the beta version of RipMe 2.1.10. All the Tumblr links tested the same thing happens to me. I have tried flickr, in case the program doesn't work, and if it works the first time. What else needs to be done or is Tumblr blocked on all its blogs?

Downloading http://api.tumblr.com/v2/blog/iguanodont.tumblr.com/posts/photo?api_key=JFNLu3CbINQjRdUvZibXW9VpSEVYYtiPJ86o8YmvgLZIoKyuNX&offset=0
Rip complete, saved to /Volumes/LCrugged5TB/Aplicaciones/rips/tumblr_iguanodont.tumblr.com
Cannot invoke "org.json.JSONObject.getJSONObject(String)" because "json" is null
